src/Request/uapi-request.js

```diff
@@ -41,7 +47,7 @@ module.exports = function (service, auth, reqType, rootObject,
 
   return function serviceFunc(params) {
     if (debugMode) {
-      console.log('Input params ', params);
+      log('Input params ', pd.json(JSON.stringify(params)));
```

Comment:
It's better to pass JSON untouched just as `pd.json(params)`: https://github.com/vkiryukhin/pretty-data/blob/master/pretty-data.js#L137
Otherwise it will be parsed and stringified again.